packages/@aws-cdk/toolkit-lib/test/api/drift/drift.test.ts

```diff
@@ -0,0 +1,640 @@
+import type * as cxapi from '@aws-cdk/cx-api';
+import type { DescribeStackResourceDriftsCommandOutput, StackResourceDrift } from '@aws-sdk/client-cloudformation';
+import {
+  DescribeStackDriftDetectionStatusCommand,
+  DescribeStackResourceDriftsCommand,
+  DetectStackDriftCommand,
+  DetectStackResourceDriftCommand,
+} from '@aws-sdk/client-cloudformation';
+import { detectStackDrift, DriftFormatter } from '../../../lib/api/drift';
+import { IoHelper } from '../../../lib/api/io/private';
+import { ToolkitError } from '../../../lib/toolkit/toolkit-error';
+import { mockCloudFormationClient, MockSdk } from '../../_helpers/mock-sdk';
+
+jest.mock('../../../lib/api/io/private', () => {
+  const originalModule = jest.requireActual('../../../lib/api/io/private');
+  return {
+    ...originalModule,
+    IO: {
+      ...originalModule.IO,
+      DEFAULT_TOOLKIT_DEBUG: {
+        msg: jest.fn().mockReturnValue('mocked-message'),
+      },
+    },
+  };
+});
```

Comment:
There is absolutely no need to mock this module. Look for `TestIoHost` if you need to assert messages.